x-pack/plugins/security_solution/public/detections/components/rules/pre_packaged_rules/load_empty_prompt.tsx

```diff
@@ -80,14 +67,13 @@ const PrePackagedRulesPromptComponent: React.FC<PrePackagedRulesPromptProps> = (
         <EuiFlexGroup justifyContent="center">
           <EuiFlexItem grow={false}>{loadPrebuiltRulesAndTemplatesButton}</EuiFlexItem>
           <EuiFlexItem grow={false}>
-            <LinkButton
+            <SecuritySolutionLinkButton
```

Comment:
A small refactor.

Reply:
This looks so much cleaner! 🙌  Thank you!